pyproject.toml

```
@@ -22,7 +22,7 @@ dependencies = [
     "numpy",
     "qiskit >= 0.45.1, < 0.46",
     "qiskit-aer >= 0.13.1, < 0.14",
-    "iqm-client >= 16.0, < 17.0"
+    "iqm-client >= 17.0, < 18.0"
```

```suggestion
    "iqm-client >= 17.1, < 18.0"
```

To avoid the possibility of using the version of IQM Client which had quantum architecture a compatibility bug.